staging/src/k8s.io/apiserver/pkg/server/filters/priority-and-fairness.go

```diff
@@ -78,6 +82,8 @@ func WithPriorityAndFairness(
 	}
 	startOnce.Do(func() {
 		startRecordingUsage(watermark)
+	})
+	apfStartOnce.Do(func() {
```

Comment:
Right. Separating `startOnce` of the two filters is used to avoid race condition when running `maxinflight_test.go` and `priority-and-fairness_test.go` together, in which case both `WithPriorityAndFairness` and `WithMaxInFlightLimit` are invoked.